[0m16:01:25.093444 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BCCC393F20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BCCF4D4D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BCCEF56120>]}


============================== 16:01:25.113886 | c143baa3-bc9d-4089-8310-974643f1ab6b ==============================
[0m16:01:25.113886 [info ] [MainThread]: Running with dbt=1.10.11
[0m16:01:25.119996 [debug] [MainThread]: running dbt with arguments {'partial_parse': 'True', 'write_json': 'True', 'send_anonymous_usage_stats': 'True', 'use_colors': 'True', 'cache_selected_only': 'False', 'target_path': 'None', 'warn_error': 'None', 'invocation_command': 'dbt init', 'log_format': 'default', 'version_check': 'True', 'introspect': 'True', 'debug': 'False', 'printer_width': '80', 'fail_fast': 'False', 'no_print': 'None', 'empty': 'None', 'use_experimental_parser': 'False', 'quiet': 'False', 'log_cache_events': 'False', 'indirect_selection': 'eager', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\chaitanya.garigipati\\.dbt', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True'}
[0m16:01:25.291703 [info ] [MainThread]: Creating dbt configuration folder at C:\Users\chaitanya.garigipati\.dbt
[0m16:01:34.280475 [debug] [MainThread]: Starter project path: C:\Users\chaitanya.garigipati\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m16:01:34.411433 [info ] [MainThread]: 
Your new dbt project "DBT_DEV" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:01:34.414941 [info ] [MainThread]: Setting up your profile.
[0m16:01:34.424496 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m16:01:34.432683 [debug] [MainThread]: Command `dbt init` failed at 16:01:34.431486 after 9.68 seconds
[0m16:01:34.438093 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BCCE856360>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BCD08073B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002BCCC2B8830>]}
[0m16:01:34.444639 [debug] [MainThread]: Flushing usage events
[0m16:01:37.706172 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m16:11:42.214594 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020B2B5D42F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020B2DBE7A40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020B2DCE77A0>]}


============================== 16:11:42.229112 | 336534de-8a51-489c-a264-bbea848357cc ==============================
[0m16:11:42.229112 [info ] [MainThread]: Running with dbt=1.10.11
[0m16:11:42.229112 [debug] [MainThread]: running dbt with arguments {'partial_parse': 'True', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'send_anonymous_usage_stats': 'True', 'version_check': 'True', 'cache_selected_only': 'False', 'target_path': 'None', 'warn_error': 'None', 'invocation_command': 'dbt init', 'log_format': 'default', 'use_colors': 'True', 'introspect': 'True', 'debug': 'False', 'printer_width': '80', 'fail_fast': 'False', 'no_print': 'None', 'empty': 'None', 'quiet': 'False', 'use_experimental_parser': 'False', 'log_cache_events': 'False', 'indirect_selection': 'eager', 'log_path': 'logs', 'write_json': 'True', 'static_parser': 'True', 'profiles_dir': 'C:\\Users\\chaitanya.garigipati\\.dbt'}
[0m16:11:47.598855 [info ] [MainThread]: A project called DBT_DEV already exists here.
[0m16:11:47.607536 [debug] [MainThread]: Command `dbt init` succeeded at 16:11:47.606533 after 5.61 seconds
[0m16:11:47.609532 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020B2DBE7A40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020B2EA501A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000020B2EC058E0>]}
[0m16:11:47.612031 [debug] [MainThread]: Flushing usage events
[0m16:11:48.755250 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m16:11:55.512436 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002265188C3E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000226543C85F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000226523F4410>]}


============================== 16:11:55.528067 | 032e39ec-bbf4-491c-b004-d474fc0a215e ==============================
[0m16:11:55.528067 [info ] [MainThread]: Running with dbt=1.10.11
[0m16:11:55.528067 [debug] [MainThread]: running dbt with arguments {'partial_parse': 'True', 'write_json': 'True', 'send_anonymous_usage_stats': 'True', 'use_colors': 'True', 'cache_selected_only': 'False', 'target_path': 'None', 'version_check': 'True', 'invocation_command': 'dbt init', 'log_format': 'default', 'warn_error': 'None', 'introspect': 'True', 'debug': 'False', 'printer_width': '80', 'fail_fast': 'False', 'no_print': 'None', 'use_experimental_parser': 'False', 'empty': 'None', 'log_cache_events': 'False', 'quiet': 'False', 'log_path': 'logs', 'indirect_selection': 'eager', 'profiles_dir': 'C:\\Users\\chaitanya.garigipati\\.dbt', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True'}
[0m16:12:04.059448 [debug] [MainThread]: Starter project path: C:\Users\chaitanya.garigipati\AppData\Local\Programs\Python\Python312\Lib\site-packages\dbt\include\starter_project
[0m16:12:04.172603 [info ] [MainThread]: 
Your new dbt project "DBT_1" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:12:04.172603 [info ] [MainThread]: Setting up your profile.
[0m16:12:51.507219 [info ] [MainThread]: Profile DBT_1 written to C:\Users\chaitanya.garigipati\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:12:51.512175 [debug] [MainThread]: Command `dbt init` succeeded at 16:12:51.511161 after 56.20 seconds
[0m16:12:51.514555 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002265453AC30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000226795DBD10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000226795DBE90>]}
[0m16:12:51.520387 [debug] [MainThread]: Flushing usage events
[0m16:12:54.767126 [debug] [MainThread]: An error was encountered while trying to flush usage events
